docs/modules/datasources/pages/performance_datasource.adoc

```diff
@@ -190,13 +190,19 @@ To query only SNMP interface resources on the node with ID 42, and to display th
 nodeResources(42, label, interfaceSnmp)
```

Comment:
Note, in line 188, please update "interfaces resources' labels" to "interface resource labels"

Reply:
shouldn't it be the `interface's resource labels` as there are multiple "resource label"s for each interface?

Reply:
I read it as a singular "SNMP interface". If there are multiple, then yes, it should be "interface's resource labels". Thanks!